lib/onc_certification_g10_test_kit/bulk_data_group_export_stu2.rb

```diff
@@ -0,0 +1,52 @@
+require_relative 'bulk_data_group_export'
+require_relative 'export_kick_off_performer'
+
+module ONCCertificationG10TestKit
+  class BulkDataGroupExportSTU2 < Inferno::TestGroup
+    title 'Group Compartment Export Tests STU2'
+    short_description 'Verify that the system supports Group compartment export.'
+    description <<~DESCRIPTION
+      Verify that system level export on the Bulk Data server follow the Bulk Data Access Implementation Guide
+    DESCRIPTION
+
+    id :bulk_data_group_export_stu2
+
+    test from: 'bulk_data_group_export-g10_bulk_data_server_tls_version'
```

Comment:
Pulling in each individual test for this Group as additional Bulk Data Group Export STU2 changes (beyond this PR, whose `_outputFormat` test is additive only) will alter specific tests. This is not the case for  Bulk Data Group Export STU1, which inherits directly from Bulk Data Group Export TestGroup.